ansible/group_vars/all.yml

```diff
@@ -96,3 +96,21 @@ dns_replicas: 1
 
 # See kube documentation for apiserver runtime config options.  Example below enables HPA, deployments features.
 # apiserver_extra_args: "--runtime-config=extensions/v1beta1/deployments=true"
+
+# The port that the Kubernetes apiserver component listens on.
+kube_master_api_port: 443
+
+# This directory is where all the additional config stuff goes
+# the kubernetes normally puts in /srv/kubernets.
+# This puts them in a sane location.
+# Editting this value will almost surely break something. Don't
+# change it. Things like the systemd scripts are hard coded to
+# look in here. Don't do it.
+kube_config_dir: /etc/kubernetes
```

Comment:
I need to template the location of certificates so contiv client can access the kube api server. Please see ansible/roles/contiv/templates/contiv.cfg.j2. That's why these variables need to be exposed. What would be the other way to do that?